### ISSUE TITLE
Cannot invoke "ghidra.app.util.bin.format.macho.dyld.DyldCacheSlideInfoCommon.getMappingFileOffset()" because "slideInfo" is null

### ISSUE THREAD

**Issue by jrmuizel** [NationalSecurityAgency/ghidra#6355]
**Describe the bug**
On macOS 14.4.1 I get the following error when trying to import a system library from `/System/Volumes/Preboot/Cryptexes/OS/System/Library/dyld/dyld_shared_cache_arm64e`

I'm able to browse the contents of dyld_shared_cache_arm64e with the file system browser but when I try to import a file I get:

```
2024-03-26 14:36:24 ERROR (AbstractWorker) Unexpected error processing job:  java.lang.NullPointerException: Cannot invoke "ghidra.app.util.bin.format.macho.dyld.DyldCacheSlideInfoCommon.getMappingFileOffset()" because "slideInfo" is null
	at ghidra.file.formats.ios.dyldcache.DyldCacheExtractor.getSlideFixups(DyldCacheExtractor.java:200)
	at ghidra.file.formats.ios.dyldcache.DyldCacheFileSystem.getByteProvider(DyldCacheFileSystem.java:95)
	at ghidra.formats.gfilesystem.FileSystemService.getByteProvider(FileSystemService.java:364)
	at ghidra.plugins.fsbrowser.FSBActionManager.ensureFileAccessable(FSBActionManager.java:950)
	at ghidra.plugins.fsbrowser.FSBActionManager.lambda$createImportAction$35(FSBActionManager.java:674)
	at docking.widgets.tree.GTree$1.run(GTree.java:1344)
	at ghidra.util.worker.AbstractWorker$JobCallback.process(AbstractWorker.java:133)
	at ghidra.util.worker.AbstractWorker$JobCallback.process(AbstractWorker.java:123)
	at generic.concurrent.ConcurrentQ$CallbackCallable.call(ConcurrentQ.java:658)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at generic.concurrent.FutureTaskMonitor.run(FutureTaskMonitor.java:76)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
```

**To Reproduce**
1. Import File
2. Choose /System/Volumes/Preboot/Cryptexes/OS/System/Library/dyld/dyld_shared_cache_arm64e
3. Choose file system import mode
4. Choose /System/Library/Frameworks/WebKit.framework/Versions/A/WebKit
5. See error

**Environment (please complete the following information):**
 - OS: 14.4.1
 - Java Version: 21.0.2
 - Ghidra Version: 11.0.1
 - Ghidra Origin: official build

**Comment by mstange** [comment]
I see this too, on macOS 14.4.

It's possible that this was fixed by https://github.com/NationalSecurityAgency/ghidra/commit/2aea201406325912d3c701a4ec325df575d3a9cc .

**Comment by ryanmkurtz** [comment]
Yes, this has been fixed in 11.0.2, which has just been released.  

However, I still don't have the new `dyld_cache_slide_info5` implemented, since AFAIK Apple has not released a spec on it yet.  So, the binary will load, but there will be a lot of wrong pointers.  I will fix that as soon as I can.

**Comment by ryanmkurtz** [comment]
The `dyld_cache_slide_info5` has been implemented b6e7c228f22a712bc1eaad9b2b3287096c6d380d.  This is currently available in `master` and will be included in the next major release (11.1).
